articles/libraries/chemistry/concepts/multireference.md

```diff
@@ -0,0 +1,134 @@
+
+The first approach, which we call sparse multi-reference wavefunctions, is effective when only a few components suffices to specify the superposition. The second component, known as unitary coupled-cluster methods, specify the state implicity through a unitary operator acting on a reference state. These states may be specified in [Broombridge schema](xref:microsoft.quantum.libraries.chemistry.schema.spec), and we also provide the functionality to manually specify these states through the chemistry library.
+
+## Sparse multi-reference wavefunction
+On one hand, multi-reference state $|\psi_{\rm {MCSCF}}\rangle$ may be specified explicitly as a linear combination of $N$-electron Slater determininants.
+\begin{align}
+|\psi_{\rm {MCSCF}}\rangle \propto \sum_{i_1 < i_2 < \cdots < i_N} \lambda_{i_1,i_2} a^\dagger_{i_1}a^\dagger_{i_2}\cdots a^\dagger_{i_N}|0\rangle.
+\end{align}
+For example, the state $\propto(0.1 a^\dagger_1a^\dagger_2a^\dagger_6 - 0.2 a^\dagger_2a^\dagger_1a^\dagger_5)|0\rangle$ may be specified in the chemistry library as follows.
+```csharp
+// Create a list of tuples where the first item of each 
+// tuple are indices to the creation operators acting on the
+// vacuum state, and the second item is the coefficient
+// of that basis state.
+var superposition = new[] {
+    (new[] {1, 2, 6}, 0.1),
+    (new[] {2, 1, 5}, -0.2) };
+
+// Create a fermion wavefunction object that represents the superposition.
+var wavefunction = new FermionWavefunction<int>(superposition);
+```
+This explicit representation of the superposition components is effective when only a few components need to be specified. One should avoid using this representation when many components are required to accurately capture the desired state. The reason for this is the gate cost of quantum circuit that prepares this state on a quantum computer, which scales at least linearly with the number of superposition components, and at most quadratically in the one-norm of the superposition amplitudes.
+
+## Unitary coupled-cluster wavefunction
+It is also possible to efficiently specify a multi-reference state that has exponentially many components. One common example is the unitary coupled-cluster wavefunction $|\psi_{\rm {UCC}}\rangle$. In this situation, we have a single-reference state, say, $|\psi_{\rm{SCF}}\rangle$. The components of the unitary coupled-cluster wavefunction is then specified implicity through a unitary operator acting on a reference state. This unitary operator is commonly written as $e^{T-T^\dagger}$, where $T-T^\dagger$ is the anti-Hermitian cluster operator. Thus
```

Comment:
please add that we are doing UCC SD at present.

Reply:
This is stated later on in line 53